_data/projects/simon-quality.yml

```
@@ -0,0 +1,11 @@
+---
+name: Air and Ground Monitoring Information System using Arduino UNO R3
+desc: The monitoring system for air temperature, soil moisture and soil ph uses the Arduino Uno R3, where this system can assess soil conditions based on certain parameters to determine the soil is in good condition during the planting process
+site: https://bellabeen.github.io
+tags:
+  - arduino
+  - iot
+
+upforgrabs:
+  name: Air and Ground Monitoring Information System using Arduino UNO R3
```

This needs to be a label in your project. I think this one is the most suitable option, but feel free to change it if there's another label:

```suggestion
  name: documentation
```

---

Okay thanks sir